config/certmanager/certificate-webhook.yaml

```
@@ -0,0 +1,20 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
+# More document can be found at https://docs.cert-manager.io
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+    app.kubernetes.io/name: managed-openshift4-agent
```

I guess this should be espejote and is a copy-paste leftover?

```suggestion
    app.kubernetes.io/name: espejote
```